cpp/csp/python/InitHelper.h

```diff
@@ -111,4 +111,19 @@ inline bool InitHelper::execute( PyObject * module )
 }
 
 }
+
+#if PY_VERSION_HEX < 0x03090000
+//PyMODINIT_FUNC in Python <3.9 doesn't export the function/make visible
+//this is required since we build with hidden visibility by default
+//https://github.com/scipy/scipy/issues/15996
+//https://github.com/mesonbuild/meson/pull/10369
+#if defined(__cplusplus)
```

Comment:
dont need this, its always cplusplus for us
Also rather than defining a new macro, can re just redefine PyMODINIT_FUNC here so that wer dont have to update all the module files ( and dont need to undo this once we drop 3.8 )

Reply:
done